src/integTest/groovy/com/readytalk/LifecycleIntegTest.groovy

```diff
@@ -10,7 +10,7 @@ class LifecycleIntegTest extends IntegrationSpec {
       apply plugin: 'com.readytalk.ci'
       apply plugin: 'java'
 
-      buildEnv.isCI = false
+      buildEnv.ci = false
```

Comment:
I did change the API on this a bit, but I think that is ok since we are not 1.0 yet :)